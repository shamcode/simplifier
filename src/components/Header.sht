<header>
    <h1>Logic expressions simplifier</h1>
</header>
