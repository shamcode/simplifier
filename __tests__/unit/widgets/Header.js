import Header from '../../../src/widgets/Header.sht';
import renderer from 'sham-ui-test-helpers';

it( 'renders correctly', () => {
    const meta = renderer( Header );
    meta.widget.update();
    expect( meta.toJSON() ).toMatchSnapshot();
} );
