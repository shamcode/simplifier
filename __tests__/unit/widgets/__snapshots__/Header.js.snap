// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "ID": "widget",
  "__data__": Object {
    "actionSequence": Array [
      "render",
      "bindEvents",
    ],
    "renderAsync": false,
    "renderDependence": Array [],
    "types": Array [],
  },
  "html": "<header><h1>Logic expressions simplifier</h1></header>",
  "nodes": Array [
    Object {
      "ID": undefined,
      "__data__": null,
      "html": null,
      "nodes": null,
      "options": null,
    },
  ],
  "options": Object {
    "actionSequence": Array [
      "render",
      "bindEvents",
    ],
    "renderAsync": false,
    "renderDependence": Array [],
    "types": Array [],
  },
}
`;
